packages/edit-site/src/components/global-styles/font-library-modal/style.scss

```diff
@@ -135,3 +135,10 @@ button.font-library-modal__upload-area {
 		max-width: 400px;
 	}
 }
+
+.font-library-modal__search {
```

Comment:
We can remove this. Part of the goal of using components is to eliminate local CSS, and have one "language". Besides, the gray search is part of the original designs: https://github.com/WordPress/gutenberg/issues/45271

Reply:
Thanks! I wasn't sure what to do here and thought it was safer to match the current styles. Removed this CSS in https://github.com/WordPress/gutenberg/pull/59589/commits/c301cbaa369f519ba8286c8a9fe31c85b8dc6977.